modules/extension/xsd/xsd-core/src/test/java/org/geotools/xml/SchemasTest.java

```diff
@@ -95,7 +95,8 @@ protected void setUp() throws Exception {
                 "targetNamespace='http://geotools.org/test'> " + 
             "</xsd:schema>";
         write( f, xsd );
-            
+
+        System.setProperty(Schemas.FORCE_SCHEMA_IMPORT, "false");
```

Comment:
@ridethepenguin please also add this System.setProperty statement to tearDownl test method order is undefined and, if testForcedSchemaImport runs last, the system property setting would leak into other tests because one JVM is used to run all tests in this module.